### PR TITLE
Looping biotsav

### PR DESCRIPTION
**Pull request by GaryQ-physics**
pull req

### CODE REVIEW

File: physics/biot_savart_spacepy.py [hunk 1]
```diff
@@ -39,56 +39,26 @@
...
-
-    #filename_split = events.findfile(time)
+    events_array = events.events(time[0:5])
+    print(events_array.shape)
+    if events_array.size == 0:
+        print('hello there')
+        continue
 
-    filename = conf['run_path'] + filename_split
+    filename = conf['run_path'] + filename_split[0:34] # remove '.cdf'
```
Comment (rweigel) [inline physics/biot_savart_spacepy.py]: Won't work if filename changes. Use `replace` or https://kite.com/python/examples/4300/os-remove-the-file-extension-from-a-filename

File: physics/biot_savart_spacepy.py [hunk 2]
```diff
@@ -1,15 +1,44 @@
...
-opening: /home/gary/magnetosphere/data/SCARR5_GM_IO2/IO2/3d__var_3_e20031120-174500-000.out
----------
-dBmhd = 1004.0975310504954
-fractional error = -0.24662938647340985
+Reading /home/gary/magnetosphere/data/SCARR5_GM_IO2/IO2/ls-1.txt
+Read /home/gary/magnetosphere/data/SCARR5_GM_IO2/IO2/ls-1.txt
+(117, 8)
+opening: /home/gary/magnetosphere/data/SCARR5_GM_IO2/IO2/3d__var_3_e20031120-070000-000.out
+---
+Event: 2003,11,20,7,0;55.00,241.00
```
Comment (rweigel) [inline physics/biot_savart_spacepy.py]: Print out info about how the calculation was made. `dBmhd` is `dB` in SWMF output, etc. When reading the log file, I should be able to understand what method you were using to compute dB.

Comment (rweigel) [reply physics/biot_savart_spacepy.py]: You can avoid having to copy/paste "typical output" by running the following on the command line

```
python biot_savart_spacepy.py > biot_savart_spacepy.log
```

Then commit the log file and reference it in your comment.

Comment (GaryQ-physics) [reply physics/biot_savart_spacepy.py]: when trying this, the resulting biot_savart_spacepy.log appears to be a blank document.

Comment (rweigel) [reply physics/biot_savart_spacepy.py]: This is somewhat unexpected.

`python biot_savart_spacepy.py` should result in text being shown in the terminal. When you use `>`, it sends everything you see on the terminal to the file that follows.

Look into `stdout` and `stderr` to learn more about the `>` and `|` shell operators.

Comment (GaryQ-physics) [reply physics/biot_savart_spacepy.py]: ok now I found it. It looks like they make a blank copy and put it in a different folder

**Pull request merged**